src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+interface Props {
+  post: Post,
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      {post.user && (
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={post.user} />
+        </p>
+      )}
```

Comment:
You render `<p> {'posted by'} </p>` only if you have user. 

Maybe would be better move this to `userinfo`